.github/CODEOWNERS

```diff
@@ -1,247 +1,309 @@
+
+# Utilities that support central developer workflow infrastructure.
+/utility/src/*/java/org/oppia/android/util/platformparameter/ @oppia/android-developer-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/system/ @oppia/android-developer-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/threading/ @oppia/android-developer-infrastructure-reviewers
+
+# Utilities needed by the domain layer of the app.
+/utility/src/*/java/org/oppia/android/util/caching/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/data/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/gcsresource/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/locale/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/logging/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/math/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/networking/ @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/parser/image @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/parser/math @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/parser/svg @oppia/android-app-infrastructure-reviewers
+/utility/src/*/java/org/oppia/android/util/profile/ @oppia/android-app-infrastructure-reviewers
+/utility/src/test/res/values/strings.xml @oppia/android-app-infrastructure-reviewers
+
+# Utilities that are primarily used for frontend/UI purposes.
+/utility/src/*/java/org/oppia/android/util/accessibility/ @oppia/android-frontend-reviewers
+/utility/src/*/java/org/oppia/android/util/statusbar/ @oppia/android-frontend-reviewers
+/utility/src/*/java/org/oppia/android/util/extensions/ @oppia/android-frontend-reviewers
+/utility/src/*/java/org/oppia/android/util/parser/html @oppia/android-frontend-reviewers
+/utility/src/*/res/**/*.xml @oppia/android-frontend-reviewers
+
+################################################################################
+#                                     Wiki                                     #
+################################################################################
+
+# Wiki pages are shared among all owners.
+/wiki/ @oppia/owners @oppia/android-app-infrastructure-reviewers @oppia/android-developer-infrastructure-reviewers @oppia/android-frontend-reviewers
+
+
+
+# ---====================== REPOSITORY CONFIGURATIONS =====================--- #
+
+
+
+################################################################################
+#                           Git/GitHub configurations                          #
+################################################################################
+
+# Top-level GitHub and Git configuration files.
+/.gitignore @oppia/owners
```

Comment:
Wouldn't these all be dev-workflow, like the stuff under "sensitive files" below? I would reserve `owners` for things that only the repo admins should modify.

It might be that "owners" is a temporary substitute for @seanlip, since I can review dev workflow stuff but not dev tooling. In that case, I think the overall category should still be `android-dev-workflow-reviewers`, but for now you might want to temporarily have an `android-devtools-reviewers` team for the bazel/kotlin stuff. Later, when things are stable enough for the dev workflow team to include those in their remit (since they should be ensuring the build,/onboarding/etc. processes are streamlined for contributors), we can combine those into a single `android-dev-workflow-reviewers` team.

Reply:
This wasn't meant to be a substitute for you, actually. The thinking here is that these files should rarely change and, when they do, there should be a good reason that the team's technical leadership approve. We're mainly talking about the README, PR/issue templates, and .gitignore. All of these files need to be carefully reviewed and often require context that exceeds the purview of any of the individual reviewer teams.

That being said, I'm not opposed with making the README & gitignore special one-offs for owners reviews, and making everything else android-dev-workflow-reviewers since I think it's not unreasonable for them to own the issue & discussion templates.

It also occurred to me that .github/ was not meant to be as much of a blanket pattern as it is here, so that's been fixed as well. Hopefully this actually better helps clarify the distinction between these two types of files.

Reply:
No follow-up comment and you mentioned LGTM @seanlip, so resolving this thread.